e2e/test/iothub/messaging/MessageSendE2EPoolAmqpTests.cs

```diff
@@ -132,9 +132,6 @@ public async Task Message_IoTHubSak_DeviceSendSingleMessage_MultipleConnections_
                 (Client.Message testMessage, string payload, string p1Value) = MessageSendE2ETests.ComposeD2cTestMessage();
                 s_log.Trace($"{nameof(MessageSendE2EPoolAmqpTests)}.{testDevice.Id}: messageId='{testMessage.MessageId}' payload='{payload}' p1Value='{p1Value}'");
                 await deviceClient.SendEventAsync(testMessage).ConfigureAwait(false);
```

Comment:
Is there some logic built into the SDK that makes this method throw an exception if IotHub responds to this message with a non-200 status code?

Reply:
If the `.SendEventAsync()` call throws an exception, the SDK will retry based on it's retry policy, or if non-retryable, will forward the exception to the calling application.